dataline-integrations/singer/local_csv/destination/entrypoint.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -e
+
+PROCESSED_CONFIG_FILE="processed_config.json"
+
+function echo2() {
+  echo >&2 "$@"
+}
+
+function error() {
+  echo2 "$@"
+  exit 1
+}
+
+function main() {
+  echo "hi!"
+  ARGS=
+  while [ $# -ne 0 ]; do
+    case "$1" in
+    -a | --discover)
+      ARGS="$ARGS --discover"
+      shift 1
+      ;;
+    -b | --config)
+      jq '.destination_path = "/local/" + .destination_path' $2 > $PROCESSED_CONFIG_FILE
```

Comment:
does it cause error if destination_path is already prefixed with '/'?

Reply:
no `/local//whatever/they/input` is still a legal path. the extra `/` gets allided.